### DISCUSSION TITLE
how to select several marker based on radius polygon draw and show the information from its dataframe

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#568] [Q&A] [answered]
Dear All,

is there anyone know how to select several markers based on radius polygon draw in leafmap and show the information from its dataframe?

thanks

**Comment by giswqs** [comment, marked as answer]
You can use `m.user_rois` to retrieve the drawn shapes as a FeatureCollection, then use it to intersect the marker layer using GeoPandas.

**Comment by rh-afk** [comment]
hi @giswqs , did you have example some where probably in leafmap website or some where so I can follow and try to understand. Currently I try to implement in solara. draw polygon and get information coordinate, select data based on that coordinate draw in map and filter the dataframe
